server/src/main/java/org/elasticsearch/search/fetch/subphase/DocValueFieldsFetchSubPhase.java

```diff
@@ -158,5 +156,12 @@ public void hitsExecute(SearchContext context, SearchHit[] hits) throws IOExcept
                 }
             }
         }
+
+        if (noFormatFields.isEmpty() == false) {
+            DEPRECATION_LOGGER.deprecated("There are doc-value field which are not using a format. The output will "
+                    + "change in 7.0 when doc value fields get formatted based on mappings by default. It is recommended to pass "
+                    + "[format={}] with a doc value field in order to opt in for the future behaviour and ease the migration to "
+                    + "7.0: [{}]", DocValueFieldsContext.USE_DEFAULT_FORMAT, noFormatFields);
```

Comment:
do we really need the outer brackets, there will be double brackets in the string because of the list, right?